### DISCUSSION TITLE
Can someone make a demo for this package to show up how that works?

### DISCUSSION THREAD

**Discussion by stojankukrika** [joshhanley/livewire-autocomplete#21] [Help]
I really like to add this package if it works well and has an option for example enter text input and can select using keyboard items from autocomplete. Any video demo or something like that would help.

**Comment by joshhanley** [comment]
@stojankukrika there is already a demo app you can play with, see here in the readme https://github.com/joshhanley/livewire-autocomplete#demo-app

Hope it helps!

**Reply by stojankukrika** [reply]
Thank you for the link but this repo does not work. I just clone it and follow the install instructions there with additional `yarn install` and `npm run dev` but only I get a black page like this one:
<img width="1706" alt="Screenshot 2023-03-01 at 22 59 32" src="https://user-images.githubusercontent.com/10199584/222275234-199307fe-8735-44d4-87ae-a77025572759.png">

**Reply by joshhanley** [reply]
@stojankukrika hmm, I just pulled the repo myself and followed the instructions and it works happily for me. You should not be running `yarn install` or `npm run dev` though, not sure why you did that. Try it without doing it and see if that works.

**Reply by stojankukrika** [reply]
Without that I get this error:
<img width="1668" alt="Screenshot 2023-03-01 at 23 23 18" src="https://user-images.githubusercontent.com/10199584/222278985-b1572dc8-e4c1-4aca-8415-b2d6049d32b5.png">

**Reply by joshhanley** [reply]
@stojankukrika ok that's weird! Can you confirm, did you run `composer install` or `composer update`? It should be install, just to make sure the versions are the same (for the demo).

Can I suggest deleting it and trying again? The only step that was missing in the instructions for me was running `php artisan key:generate` after the `.env` has been copied and configured. But other than that, it all ran happily following those instructions directly.

**Reply by stojankukrika** [reply]
Yes, I run that command as well `php artisan key:generate` because I see that is missing. I have never before gotten this kind an error so I am not sure what is it means. And yes, I have valet and PHP 8.0 on my local.

**Reply by joshhanley** [reply]
@stojankukrika yeah strange, I'm running valet and PHP 8.1 too, so shouldn't be an issue with your setup.

**Comment by joshhanley** [comment]
I just realised I didn't answer your original question though, yeah it does support keyboard navigation, so you can start typing to search and arrow down/up to highlight an option and hit enter to select 🙂

**Reply by stojankukrika** [reply]
if I make it works :)
